doc/modules/model_evaluation.rst

```diff
@@ -1109,6 +1110,71 @@ function:
     >>> matthews_corrcoef(y_true, y_pred)  # doctest: +ELLIPSIS
     -0.33...
 
+.. _multilabel_confusion_matrix:
+
+Multi-label confusion matrix
+----------------
+
+The :func:`multilabel_confusion_matrix` function computes multilabel confusion
+matrix class-wisely (default) or sample-wisely (samplewise=True) to evaluate
+the accuracy of a classification. multilabel_confusion_matrix also treats
+multiclass data as if it were multilabel, as this is a transformation commonly
+applied to evaluate multiclass problems with binary classification metrics
+(such as precision, recall, etc.).
+
+When calculating class-wise multilabel confusion matrix :math:`MCM`, the
+count of true negatives for class :math:`i` is :math:`MCM_{i,0,0}`, false
+negatives is :math:`MCM_{i,1,0}`, true positives is :math:`MCM_{i,1,1}`
+and false positives is :math:`MCM_{i,0,1}`.
+
+Here is an example demonstrating the use of the
```

Comment:
I wonder if it would be useful to illustrate how to calculate something like specificity for each class after these basic usages.

Reply:
Yeah, of course, I will add them. For now, only examples, not metrics or anything else, is that right?

Reply:
IMO, yes